examples/DynaSwing/description/DynaSwingExamples.tex

```diff
@@ -607,6 +607,52 @@ \section{Results}
   \caption{Active and reactive powers in pu}
 \end{figure}
 
+
+\chapter{Grid Forming - Grid Following}
+
+This chapter presents a test case that simulates a 5-machine all-converter-interfaced power system. Two converters are grid following and use the WECC WTG4A model presented above and three converters are grid forming converters using a droop control.
+
+\section{Test case description}
+
+\par The test case is presented in the following figure. INF, HVDC1 and HVDC2 are grid forming converters while WP1 and WP2 are wind plants using the grid following WECC WTG4A model.
+
+\begin{figure}[H]
+  \includegraphics[width=\textwidth]{GFMGFL.png}
+  \caption{Test case diagram}
+\end{figure}
+
+\subsection{Models}
+
+\par The models used for the converters and their control are fully described in the \Dynawo library.
+
+\subsection{Solver}
+
+\par The solver used is the simplified solver SIM with the following parameters:
+
+\begin{itemize}
+\item $hMin$=$10^{-4}$;
+\item $hMax$=$0.003$;
+\item $minimalAcceptableStep$=$10^{-6}$.
+\end{itemize}
+
+\subsection{Scenario}
+
+\par First, an active power double step is played at WP2 at t=1s and t=2s. Then a line is disconnected at t=3s. Then an active power double step is played at HVDC1 at t=4s and t=5s. Finally, a fault is played at t=7s and cleared at t=7.1s.
+
```

Comment:
I'm always not very fan to have all the events in one single simulation instead of several ones (maybe at least for the fault).

Reply:
So you suggest having several test cases ?

Reply:
Yes

Reply:
Ok